packages/components/card/src/asset-card/AssetCard.styles.ts

```diff
@@ -0,0 +1,42 @@
+import { css } from 'emotion';
+import tokens from '@contentful/f36-tokens';
+
+export const getAssetCardStyles = () => {
+  return {
+    contentBody: css({
+      padding: 0,
+    }),
+    header: css({
+      alignItems: 'center',
+      borderBottomColor: tokens.gray200,
+      borderBottomStyle: 'solid',
+      borderBottomWidth: 1,
+      borderTopLeftRadius: tokens.borderRadiusMedium,
+      borderTopRightRadius: tokens.borderRadiusMedium,
+      boxSizing: 'border-box',
+      color: tokens.gray600,
+      fontSize: tokens.fontSizeM,
+      fontWeight: tokens.fontWeightNormal,
+      gridColumn: 'content',
+      gridRow: 'header',
+      lineHeight: tokens.lineHeightM,
+      paddingBottom: `calc(1rem * (10 / ${tokens.fontBaseDefault}))`,
```

Comment:
It would be nice to repalce all properties that use calculated spacing value (using fontBaseDefault) with the closest spacing token.

Reply:
@fabe @domarku can we do that? 🙂

Reply:
Yes! Let's use `spacing-xs` here. It will reduce the height of the header section to `36px`, which is a good thing because it will contribute to items taking less vertical real-estate.

The only downside is that the "more" button's height will go down `4px`. I don't think that's terrible as it will still be close to `40px` in the size of its click-area.